pkg/http/http.go

```diff
@@ -113,18 +132,35 @@ func (c *HTTPClient) setAuth(req *http.Request) {
 }
 
 func (c *HTTPClient) MakeRequest(ctx context.Context) (*http.Response, error) {
```

Comment:
Please add test case for covering POST method

Reply:
Done